src/modules/shared/components/MediaCardList/MediaCardList.const.ts

```diff
@@ -6,3 +6,8 @@ export const MEDIA_CARD_LIST_GRID_BP_COLS = {
 	[Breakpoints.md]: 2,
 	[Breakpoints.sm]: 1,
 };
+
+export const RESULTS_MAX = 10000;
+export const ITEMS_PER_PAGE = 39;
```

Comment:
use the exisiting const instead of redeclaring this one: VISITOR_SPACE_ITEM_COUNT